docker-compose.yml

```diff
@@ -70,7 +71,7 @@ services:
     volumes:
       - ./scripts:/code/scripts
       ## Comment to make bcoin chain and wallet database NOT persistant
-      - ~/.bcoin_bpanel:/code/.bcoin
+      - ~/.bcoin:/root/.bcoin
```

Comment:
This didn't feel great to me, but I couldn't use something like `$HOME/.bcoin` which I need for bcfg in bcoin-init.js to work properly. @Thann @tynes let me know if you have any ideas.